src/components/WishlistPage/WishlistItem/WishlistItem.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import cx from 'classnames';
+
+import './WishlistItem.scss';
+
+const CN = 'wishlist-item';
+
+const WishlistItem = ({ item, removeFromWishlist }) => {
+  const { photo, title, collection, category } = item;
+
+  return (
+    <div className={`${cx(CN)}`}>
+      <img className={`${CN}__image`} src={photo} alt={title}></img>
+      <h3 className={`${cx(CN)}__title`}>{title}</h3>
+      <h4 className={`${cx(CN)}__subtitle`}>{collection}</h4>
+      <h4 className={`${cx(CN)}__subtitle`}>{category}</h4>
+      <button className={`${CN}__button`}>
+        <div className="col-2"><i className="cart small icon"/></div>
+        <div className="col-8 col-right"><h4>Add to cart</h4></div>
+      </button>
+      <button onClick={removeFromWishlist.bind(null, item)} className={`${CN}__button ${CN}__button--remove`}>
```

Comment:
Why are you bind context to `removeFromWishlist`?

Reply:
It's not a good idea to use bind method inside the render. You can create function called `onClick` and inside it you would have same access to `item` and `removeFromWishlist ` and you will call it inside. And in props you will pass just <button onClick={onClick} className={`${CN}__button ${CN}__button--remove`}>